models/pagination.js

```diff
@@ -1,4 +1,4 @@
-async function get({ total_rows, page, per_page, strategy }) {
+function get({ total_rows, page, per_page, strategy }) {
   const firstPage = 1;
   const lastPage = Math.ceil(total_rows / per_page);
```

Comment:
Que tal aproveitarmos para corrigir `lastPage`?

Hoje retorna `0` quando não há resultados, mas acredito que deveria retornar `1`:

```suggestion
  const lastPage = Math.ceil(total_rows / per_page) || 1;
```